jax/interpreters/ad.py

```diff
@@ -575,9 +583,9 @@ def nonzero_outputs(*args, **kwargs):
   yield results, [type(r) is not Zero for r in results]
 
 
-def map_transpose(primitive, params, call_jaxpr, args, ct, _):
+def map_transpose(primitive, params, call_jaxpr, args, ct, _, reduce_axes):
```

Comment:
An interesting benchmark for this change would be to modify the code in map_transpose to stop doing the reduction by itself and instead delegate to `backward_pass`! But we can also do that as a follow-up if you want

Reply:
I think we have to do this in a follow-up because map primitives (i.e. `pmap`) don't use avals-with-names yet! We should certainly delegate to `backward_pass` inside `xmap_transpose` (which doesn't exist yet, and which requires such delegation/this change for efficiency when there's broadcasting in the forward pass and vectorization in the schedule).